gobblin-core/build.gradle

```diff
@@ -50,6 +50,8 @@ dependencies {
 
   runtime externalDependency.protobuf
 
+  runtime files('./src/main/resources/lib/ojdbc6.jar')
```

Comment:
@lbendig, I removed the jar from `./src/main/resources/lib/ojdbc6.jar`, and I removed this dependency for my upcoming push. You mention that the jars can be passed in `goblin-mapreduce.sh`. Is the same true for `gobblin-standalone.sh`